### DISCUSSION TITLE
LLM Math

### DISCUSSION THREAD

**Discussion by yohanespm77** [teamalgoritma/community#741] [Q&A] [answered]
Saya mencoba menjalankan kode saat DSS LLM, seperti berikut : 
<img width="749" alt="image" src="https://github.com/teamalgoritma/community/assets/106010284/e2aab7f0-25da-4ef7-bb74-b883923ad3c4">

Namun saat saya mengajukan pertanyaan matematika, terdapat error : 
<img width="699" alt="image" src="https://github.com/teamalgoritma/community/assets/106010284/b4fbad13-e680-45cb-bc48-3010469e491a">

Mohon bantuannya kak, terima kasih 🙏

**Comment by IrfanChairurrachman** [comment, marked as answer]
Dear @yohanespm77 ,

Jika dilihat dari errornya adlaah `VariableNode object is not callable. Please try again with a valid numerical expression` dimana LLM tidak dapat memproses prompt yang diberikan. Jika mengacu ke diskusi ini dapat dikatakan tidak semua ekspresi matematika dapat diolah [Diskusi Github LLM](https://github.com/langchain-ai/langchain/issues/3071#issuecomment-1575495184).

Mungkin Anda dapat mengubah prompt nya atau dapat melakukan saran yang diberikan seperti di diskusi Github LLM yang telah diberikan.

Semoga informasi ini dapat membantu. Bila ternyata informasi ini dapat menjawab pertanyaan Anda, Anda dapat “Mark as answer” jawaban ini.
